docs/learning-powershell/README.md

```diff
@@ -106,7 +106,7 @@ Recommended Training and Reading
 ----
 - Microsoft Virtual Academy: [Getting Started with PowerShell][getstarted-with-powershell]
 - [Why Learn PowerShell][why-learn-powershell] by Ed Wilson
-- PowerShell Web Docs: [Basic cookbooks][basic-cookbooks]
+- PowerShell Web Docs: [Basic cookbooks][https://docs.microsoft.com/en-us/powershell/scripting/getting-started/fundamental/sample-scripts-for-administration?view=powershell-6]
```

Comment:
Please use the file pattern and move the url to eof (add `[basic-cookbooks]:url`).